lib/src/main/java/org/team100/lib/commands/drivetrain/DriveWithProfile2.java

```diff
@@ -159,7 +161,7 @@ public void execute() {
 
     @Override
     public boolean isFinished() {
-        if (!m_fieldRelativeGoal.get().isPresent()) return true;
+        if (!m_fieldRelativeGoalSupplier.get().isPresent() && m_fieldRelativeGoal == null) return true;
```

Comment:
don't you want the command to keep going even when it loses sight of the target?

Reply:
See other comment about null.